resources/sdk/pureclouddotnet/templates/README.mustache

```diff
@@ -76,6 +76,31 @@ Console.WriteLine("Access token=" + accessTokenInfo.AccessToken);
 Configuration.Default.AccessToken = accessTokenInfo.AccessToken;
 ```
 
+By default the SDK will transparently request a new access token using the refresh token when the access token expires. If you wish to implement the refresh logic set _ShouldRefreshAccessToken_ to false and store the refresh token from the auth response:
+
+```{"language":"csharp"}
+var refreshToken = accessTokenInfo.RefreshToken;
+Configuration.Default.ShouldRefreshAccessToken = false;
+```
+
+You can use the _ExpiresIn_ value to determine how long the token will live and proactively request a new one before it expires.
+
+```{"language":"csharp"}
+var tokenTimeToLive = authTokenInfo.ExpiresIn;
+```
+
+When the access token expires refresh it using the _PostToken_ method using the same clientId and clientSecret as used to request it.
+
+```{"language":"csharp"}
+var accessTokenInfo = Configuration.Default.ApiClient.PostToken("18a4c365-7ea3-4f0g-9fb7-884fb4d2e9c6",
+  "M7FfdYQyL5TA6BdbEZ8M9-Wx4uZai1rNQ7jcuFdcJJo",
+  authorizationCode: refreshToken,
+  isRefreshRequest: true);
+Console.WriteLine("Access token=" + accessTokenInfo.AccessToken);
+refreshToken = accessTokenInfo.RefreshToken;
+Configuration.Default.AccessToken = accessTokenInfo.AccessToken;
```

Comment:
This line shouldn't be necessary anymore if PostToken is mutating the configuration and setting the auth info now.